code/style.css

```diff
@@ -1,9 +1,99 @@
-/* 
-If you can see a peach background, the css file is correctly linked. 
-Feel free to change or remove this styling! 
-We want you focus A LOT on the styling for this project. 
-*/
-
 body {
-  background-color: peachpuff;
+  background-color: #c0fdff;
+  font-family: "HelveticaNeueLt", "Helvetica", sans-serif;
+}
+
+h2 {
+  font-weight: normal;
+  font-style: normal;
+  color: #6c757d;
+  margin: 15px 0px 20px 0px !important;
+}
+
+span {
+  color: #adb5bd;
+}
+
+input {
+  width: 200px;
+  margin: 6px 6px 6px 6px !important;
```

Comment:
When you have the same value for all four sides - you just have to write it once: `margin: 6px;`